playbook/app/pb_kits/playbook/pb_selectable_list/_selectable_list.scss

```diff
@@ -7,6 +7,9 @@
     &:hover {
       background-color: $bg_light;
     }
+    &[class*=checked_item] {
+      background-color: #f7fbff;
```

Comment:
@jasperfurniss @carloslimasd I noticed we're using a custom color for the checked_item class. I checked the story and there's no recommended background color to use for a checked item. Should it just be our $bg_light token. Or are we trying to differentiate a checked item vs a hovered item? Either way we should be using one of our color tokens for this. Carlos you can find our color tokens in playbook/app/pb_kits/playbook/tokens/_colors.scss

Reply:
Thank you @Israel-Molestina. It is done.